index.html

```diff
@@ -59,6 +59,170 @@ <h2 class="primary-header green">
+          <header class="card-header"></header>
+          <div class="card-content">
+            <h3 class="project-title">Multi-Post Stories Gain+Glory</h3>
+            <p class="project-tags">
+  
+            <ul class="project-tags-list">
+              <li><button class="tag-button">Ruby on rails</button></li>
+              <li><button class="tag-button">css</button></li>
+              <li><button class="tag-button">Javascript</button></li>
+              <li><button class="tag-button">html</button></li>
+            </ul>
+            </p>
+  
+            <button class="green-button" id="detail-button">See Project</button>
+          </div>          
+        </article>
+      </div>
+      
+    </section>
+
+    <section class="headline" id="about-me">
+      <div class="about-me-intro">
+        <h1 class="primary-header">About me</h1>
+        <p id="about-me-paragraph">
+          I can help you build a product , feature or website Look through some
+          of my work and experience! If you like what you see and have a project
+          you need coded,
+          don’t hestiate to contact me.
+        </p>
+        <button class="green-button" id="resume-button">Get My Resume</button>
+      </div>
+      <article class="about-me-item">
+        <header><img class="about-me-items-image" src="assets/images/icon-languages.svg" alt="Language"></header>
+
+        <p id="about-me-languages">Languages</p>
+        <ul class="about-me-tags-list">
+          <li><button class="tag-button">Javascript</button></li>
+          <li><button class="tag-button">Ruby</button> </li>
+          <li><button class="tag-button">Html</button> </li>
+          <li><button class="tag-button">CSS</button> </li>
+        </ul>
+      </article>
+      <article class="about-me-item">
+        <header><img class="about-me-items-image" src="assets/images/icon-frameworks.svg" alt="Frameworks"></header>
+
+        <p id="about-me-frameworks">Frameworks</p>
+        <ul class="about-me-tags-list">
+          <li><button class="tag-button">Boostrap</button></li>
+          <li><button class="tag-button">Ruby on rails</button> </li>
+          <li><button class="tag-button">RSpec</button> </li>
+          <li><button class="tag-button">Capybara</button> </li>
+          <li><button class="tag-button">Selenium</button> </li>
+        </ul>
+      </article>
+      <article class="about-me-item">
+        <header><img class="about-me-items-image" src="assets/images/icon-skills.svg" alt="Skills"></header>
+
+        <p id="about-me-languages">Skills</p>
+        <ul class="about-me-tags-list">
+          <li><button class="tag-button">Codekit</button></li>
+          <li><button class="tag-button">GitHub</button> </li>
+          <li><button class="tag-button">Codepen</button> </li>
+          <li><button class="tag-button">GitLab</button> </li>
+          <li><button class="tag-button">Terminal</button> </li>
+        </ul>
+      </article>
```

Comment:
- [ ] to be semantically correct The technology categories ("Languages", "Frameworks" and "Skills") should be in a `ul`, each of them in a `li`, kindly fix that